src/store/store.ts

```diff
@@ -1,11 +1,11 @@
-import { createStore } from "redux";
-import reducer from "./from-service/reducer";
-import { combineReducers } from "redux";
-import thunk from "redux-thunk";
-import { applyMiddleware } from "redux";
+import { createStore } from 'redux';
+import reducer from './form-service/reducer';
+import { combineReducers } from 'redux';
+import thunk from 'redux-thunk';
+import { applyMiddleware } from 'redux';
 
 const rootReducer = combineReducers({
-  weathers: reducer,
+	weathers: reducer,
```

Comment:
weathers: reducer? a u seriously? weatherReducer

Reply:
think about if u will have more than 1 service how it will named? reducer1 reducer2 etc?

Reply:
done